cmd/create/accountroles/cmd.go

```diff
@@ -32,6 +32,11 @@ import (
 	"github.com/openshift/rosa/pkg/rosa"
 )
 
+const (
+	route53RoleArnFlag     = "route53-role-arn"
```

Comment:
@hunterkepley / @gdbranco For my understanding: Is it expected that the user creates these roles somewhere else to get the ARNs or are we waiting on Managed Policies from AWS?

Reply:
They are expected to create it in adhoc manner for the moment

Reply:
Yup- this will eventually be replaced, but for release, we are expecting adhoc roles

Reply:
Thanks for the explanation.